prime-router/docs/playbooks/shutdown.md

```diff
@@ -0,0 +1,29 @@
+# Shutdown Options
+
+In the event the project needs to shutdown quickly in the response to an incident, the following options are available:
+
+## Disable a Function
+
+If the scope of the shutdown can be limited to one or more Azure Functions, Functions can individually be disabled. To do this:
+
+1. Login to Azure Portal
+2. Navigate to `*-functionapp` > `Functions` within the Portal
+3. Click on the affected function(s)
+4. From the top menu, select the `Disable` option
+5. Repeat for each affected function
+
+## Disable HTTP Traffic
```

Comment:
See above, we should also show how to do this in the CLI:

```
az network front-door backend-pool delete --front-door-name <FD_NAME>
                                          --name <BE_POOL_NAME>
                                          --resource-group <RESX_GROUP>
```

https://docs.microsoft.com/en-us/cli/azure/network/front-door/backend-pool?view=azure-cli-latest#az_network_front_door_backend_pool_delete

Reply:
Done!